use-cases/computer_vision/1-metastases-detection-train-model.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>Why do we have to pin sagemaker and boto3 to these versions? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed this so that we don't pin specific versions. Ali has the code and will submit a new pull request